cmd/enterprise-portal/internal/subscriptionsservice/adapters_test.go

```diff
@@ -145,6 +145,30 @@ func TestConvertSubscriptionToProto(t *testing.T) {
   "salesforce": {
     "subscriptionId": "sf_sub_id"
   }
+}`),
+	}, {
+		name: "with instance category",
+		sub: &subscriptions.SubscriptionWithConditions{
+			Subscription: subscriptions.Subscription{
+				ID:             "subscription_id",
+				InstanceDomain: pointers.Ptr("sourcegraph.com"),
+				InstanceType:   pointers.Ptr("CATEGORY_PRIMARY"),
```

Comment:
use a real instance category so that it is included in the expected test output
```suggestion
				InstanceType:   pointers.Ptr("ENTERPRISE_SUBSCRIPTION_INSTANCE_TYPE_PRIMARY"),
```

Reply:
ah you're right, fixed!